kfp_notebook/pipeline/_notebook_op.py

```diff
@@ -65,6 +68,9 @@ def __init__(self,
           requirements_url: URL to a python requirements.txt file to be installed prior to running the notebook
           bootstrap_script_url: URL to a custom python bootstrap script to run
           emptydir_volume_size: Size(GB) of the volume to create for the workspace when using CRIO container runtime
+          cpu_request: number of CPUs requested for the operation
+          mem_request: memory requested for the operation (in Gi)
+          gpu_request: number of GPUs requested for the operation
```

Comment:
Seems like this needs to be updated as well.  Not sure how the wording for a 'limit' should go.  Perhaps something like "_maximum number of GPUs allowed for the operation_"?